students/eowyn/session05/mailroom.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env python
+
+import sys
+
+''' global variables'''
+donor_names = ["Margaret Atwood", "Fred Armisen",
+               "Heinz the Baron Krauss von Espy"]
+donations = [[300, 555], [240, 422, 1000], [1500, 2300]]
+DONORS = dict(zip(donor_names, donations))
+
+
+
+def safe_input():
```

Comment:
if you're going to use this --the checks for the Exceptions should be in this function -- then you can jsut replace "input" with "safe_input" everywhere you want to use it.

-CHB